experiments/pipeline.jl

```diff
@@ -0,0 +1,43 @@
+# experiment-specific Project.toml instantiation, function extensions and config load
+
+using Pkg
+experiment_id = ARGS[1]
+Pkg.activate("experiments/$experiment_id")
+Pkg.instantiate()
+using CalibrateAtmos
+pkg_dir = pkgdir(CalibrateAtmos)
+experiment_path = "$pkg_dir/experiments/$experiment_id"
+include("$experiment_path/model_interface.jl")
+# include("$experiment_path/generate_truth.jl")
```

Comment:
Is this the synthetic data for the perfect-model problem ? Should this be uncommented for general use ? (I suppose if such a dataset exists then we don't need to regenerate it).

Reply:
Oops yes, that's from when I was testing the atmos case. Good catch!